mantidproject/mantid
# Apply event time to can

Fixes #14824 
# For testing

Find the required files here: \olympic\Babylon5\Scratch\Anton\Testing\14824_apply_event_to_ can
The data files are

```
   Type        Scattering               Transmission         Direct
   SAMPLE      28797                    28808                 28784
   CAN         28793                    28823                 28784
```

The instrument is SANS2DTUBES

**Test standard reduction**
1. Load the user file
2. Set the data files and load them 
3. Run 1D Reduce
4. Confirm that no errors appear

**Test reduction with time slices**
1. Load the user file
2. Set the data files and load them 
3. In the line edit filed next to the "Time slicing" button type "5-10" (without quotation marks)
3. Run 1D Reduce
4. Confirm that no errors appear
